test/test_p2p.py

```diff
@@ -26,10 +32,19 @@ async def divide_slowly(self, top, bottom):
                 await asyncio.sleep(0.1)
                 return self.divide(top, bottom)
 
-        self.test_object = self.server.add_object('/test', TestObject())
+            @bus.Object.signal('is')
+            def everything_changed(self, level, info):
+                pass
```

Comment:
Also not a fan of this, for the same reason as above.  I wonder if it would look so awful to write something like
```python
class Object:
    everything_changed = bus.Object.signal('is')
```

or so

and a similar idea for properties.

Reply:
Yes, now that I have learned about descriptors, this looks like the best idea.